hash/nes.xml

```diff
@@ -81162,18 +81237,33 @@ be better to redump them properly. -->
 		</part>
 	</software>
 
-	<software name="mc_9x7ps" supported="no">
+	<software name="mc_9x7kd">
+		<description>9999999 in 1 (KD-6037-B)</description>
+		<year>19??</year>
+		<publisher>&lt;pirate&gt;</publisher>
+		<part name="cart" interface="nes_cart">
+			<feature name="slot" value="bmc_hp898f" />
+			<dataarea name="prg" size="131072">
+				<rom name="9999999-in-1 (kd-6037-b).prg" size="131072" crc="355f6bda" sha1="0e2614ad754c40151113ffbe8665a1743935a1c9" status="baddump" />
+			</dataarea>
+			<dataarea name="chr" size="65536">
+				<rom name="9999999-in-1 (kd-6037-b).chr" size="65536" crc="3b1bc341" sha1="5dc10cede705d1b0cc41dec5f0f0adbeefe6f0bf" status="baddump" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="mc_9x7ps">
 		<description>Prima Soft 9999999-in-1 (02 8807870-3)</description>
 		<year>19??</year>
-		<publisher>&lt; pirate&gt;</publisher>
+		<publisher>&lt;pirate&gt;</publisher>
 		<part name="cart" interface="nes_cart">
 			<feature name="slot" value="bmc_hp898f" />
 			<feature name="pcb" value="BMC-HP898F" />
 			<dataarea name="chr" size="65536">
 				<rom name="prima soft 9999999-in-1 (02 8807870-3)(unl)[u].chr" size="65536" crc="51110e59" sha1="84a8d35823265f9fc882698d1c9df44006e48970" offset="00000" status="baddump" />
 			</dataarea>
 			<dataarea name="prg" size="131072">
-				<rom name="prima soft 9999999-in-1 (02 8807870-3)(unl)[u].prg" size="131072" crc="c25fd362" sha1="17ac7b87255bf052dadb89f18faee73e63ac1402" offset="00000" status="baddump" />
+				<rom name="prima soft 9999999-in-1 (02 8807870-3)(unl)[u].prg" size="131072" crc="4ce757db" sha1="8a46630e3c44aa745ced33c2a526e173ff102f76" offset="00000" status="baddump" />
```

Comment:
What explains this hash change?

Reply:
I guess I only mentioned that in the emailed submission. The dump is the same data as before but I went ahead and reshuffled the 16K banks according to: https://wiki.nesdev.com/w/index.php?title=NES_2.0_Mapper_319#UNIF_bank_order. As for the status of the dump in that order it sounds tentative based on the discussion on the same page: https://wiki.nesdev.com/w/index.php?title=Talk:NES_2.0_Mapper_319. The long and short of it seems to be the reordering is deduced by what is plausible based on what banks are selected by what games and by the fact that it matches other dumps of games on similar boards. The PCB photo is described as unuseful: "all there is to see is a single glob and a solder pad."